### PR TITLE
Dev2

### PR DESCRIPTION
**Pull request by JatinMehandiratta**

### CODE REVIEW

File: assignment3.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+</body>
+<script src="https://code.jquery.com/jquery-3.5.1.min.js" type="text/javascript"
+    integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" type="text/javascript"
+    integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous">
+</script>
+<script>
+    $( "#link1" ).click(function() {
+  alert( "Handler for .click() called." );
+});
```
Comment (nitesh796) [inline assignment3.html]: Always format the code.

File: form.php [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+                </div>
+                <div class="text-center mb-4 mt-4">
+                    <input type="submit" id="submitbtn" value="Submit" class="btn btn-success" disabled='disabled'>
+                </div>
+            </form>
+        </div>
+    </div>
+</body>
+<script src="https://code.jquery.com/jquery-3.5.1.min.js" type="text/javascript" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
+<script>
```
Comment (nitesh796) [inline form.php]: use external js

File: assignment3.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        <a href="https://getbootstrap.com/" id="link3">Link To Bootstrap</a><br>
+        <a href="https://getbootstrap.com/" id="link4">Link To Bootstrap</a><br>
+    </div>
+</body>
+<script src="https://code.jquery.com/jquery-3.5.1.min.js" type="text/javascript"
+    integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" type="text/javascript"
+    integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous">
+</script>
+<script>
```
Comment (nitesh796) [inline assignment3.html]: Use external Js

File: selectorTraversinga1.php [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+                        <button type="submit" class="btn btn-sm btn-primary">Submit</button>
+                    </div>
+                </div>
+        </form>
+    </div>
+</body>
+<script src="https://code.jquery.com/jquery-3.5.1.min.js" type="text/javascript" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" type="text/javascript" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous">
+</script>
+<script>
```
Comment (nitesh796) [inline selectorTraversinga1.php]: use external js

File: selectorTraversinga1.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
+
```
Comment (niteshiitj) [inline selectorTraversinga1.js:1]: copy paste code, but fine.

**Pull request merged**